recipes/json_dto/all/conanfile.py

```diff
@@ -16,51 +18,43 @@ class JsondtoConan(ConanFile):
     no_copy_source = True
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    def requirements(self):
-        self.requires("rapidjson/1.1.0")
+    def _min_cppstd(self):
+        return 14
 
-    def validate(self):
-        minimal_cpp_standard = "14"
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, minimal_cpp_standard)
-        minimal_version = {
+    @property
+    def _compilers_minimum_version(self):
+        return {
             "gcc": "5",
             "clang": "4",
             "apple-clang": "8",
             "Visual Studio": "15"
         }
```

Comment:
where is it used now?

Reply:
fixd

Reply:
Visual Studio min version is wrong, since it was building in c3i with Visual Studio 2015.

Reply:
it was 15 in the original recipe as well. the latest build fail was due the lack of an import.

Reply:
No, you have a commit which builds fine and where you don't even raise ConanInvalidConfiguration